e2e/rbd.go

```diff
@@ -263,7 +263,9 @@ var _ = Describe("RBD", func() {
 						e2elog.Logf("backend image count %d expected image count %d", len(images), 1)
 						Fail("validate backend image failed")
 					}
-					totalCount := 5
+					// Increased the count to 8 as we can test the
+					// maxsnapshotsonimage in CI
+					totalCount := 8
```

Comment:
I am confused by this, why pass `provisioner.maxSnapshotsOnImage=5` in the previous commit?

Reply:
as we are setting the `provisioner.maxSnapshotsOnImage=5` the csi driver will start flattening the rbd images once the parent image has 5 snapshots (including trash). as we need to test this one at least we need to create 5+ snapshots. hope its clear now